lib/active_record/dbt/column/yml.rb

```diff
@@ -79,7 +79,7 @@ def default_column_description
 
         # MEMO: [quote | dbt Developer Hub](https://docs.getdbt.com/reference/resource-properties/quote)
         def quote?
-          SQL_KEYWORDS.include?(column_name.upcase) ? true : nil
+          true if SQL_KEYWORDS.include?(column_name.upcase)
```

Comment:
https://github.com/yamotech/activerecord-dbt/actions/runs/12105212874/job/33749529541

```ruby
lib/active_record/dbt/column/yml.rb:82:62: C: [Correctable] Style/ReturnNilInPredicateMethodDefinition: Return false instead of nil in predicate methods.
          SQL_KEYWORDS.include?(column_name.upcase) ? true : nil
```